patches/0004-Update-provider-version-in-deprecation-notices.patch

```diff
@@ -109,42 +109,3 @@ index 39b679a016..b88cc5d709 100644
  
  ## Example Usage (with App Service Plan)
  
-diff --git a/website/docs/r/graph_account.html.markdown b/website/docs/r/graph_account.html.markdown
-index 95a06ce97d..924908a58f 100644
---- a/website/docs/r/graph_account.html.markdown
-+++ b/website/docs/r/graph_account.html.markdown
-@@ -10,7 +10,7 @@ description: |-
- 
- Manages a Microsoft Graph Services Account.
- 
--!> **NOTE:** This resource has been deprecated in version 3.0 of the AzureRM provider and will be removed in version 4.0. Please use [`azurerm_graph_services_account`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/graph_services_account) resource instead.
-+!> **NOTE:** This resource has been deprecated in version 5.0 of the provider and will be removed in version 6.0. Please use [`azurerm_graph_services_account`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/graph_services_account) resource instead.
```

Comment:
Those three "NOTES" are still true?

Reply:
This might likely have been removed - other docs were, but I think they missed a few.